src/github.com/getlantern/flashlight/http/http.go

```diff
@@ -81,11 +84,15 @@ func Start(httpAddr string) {
 	r.HandleFunc("/whitelist", WhitelistHandler)
 	r.HandleFunc("/proxy_on.pac", servePacFile)
 
-	assets := assetFS()
-	if assets != nil {
-		r.PathPrefix("/").Handler(http.FileServer(assets)).Methods("GET")
-		openUI()
+	start := time.Now()
+	fs, err := tarfs.New(Data, "../ui/app")
```

Comment:
@atavism This is the API for getting a filesystem.  The first parameter is a byte array (from resources.go) that contains the tar data.  The second parameter is an optional path to a local directory.  The filesystem will first search that local directory and only go to the byte array if the resource wasn't found.